src/main/java/com/gmail/yuramitryahin/Main.java

```diff
@@ -2,19 +2,36 @@
 
 import com.gmail.yuramitryahin.exception.AuthenticationException;
 import com.gmail.yuramitryahin.lib.Injector;
+import com.gmail.yuramitryahin.model.Movie;
+import com.gmail.yuramitryahin.model.MovieSession;
 import com.gmail.yuramitryahin.model.User;
 import com.gmail.yuramitryahin.service.AuthenticationService;
+import com.gmail.yuramitryahin.service.MovieService;
+import com.gmail.yuramitryahin.service.MovieSessionService;
+import com.gmail.yuramitryahin.service.ShoppingCartService;
 
 public class Main {
     private static Injector injector = Injector.getInstance("com.gmail.yuramitryahin");
+    private static final AuthenticationService authenticationService
+            = (AuthenticationService) injector.getInstance(AuthenticationService.class);
+    private static final ShoppingCartService shoppingCartService = (ShoppingCartService) injector
+            .getInstance(ShoppingCartService.class);
+    private static final MovieSessionService movieSessionService = (MovieSessionService) injector
+            .getInstance(MovieSessionService.class);
+    private static final MovieService movieService = (MovieService) injector
+            .getInstance(MovieService.class);
 
     public static void main(String[] args) throws AuthenticationException {
-        User user = new User();
-        user.setEmail("user@gmail.com");
-        user.setPassword("1234");
-        AuthenticationService authenticationService = (AuthenticationService) injector
-                .getInstance(AuthenticationService.class);
-        System.out.println(authenticationService.register(user.getEmail(), user.getPassword()));
-        System.out.println(authenticationService.login(user.getEmail(), user.getPassword()));
+        final User user = authenticationService
+                .register("user@gmail.com", "1234");
+        MovieSession movieSession = new MovieSession();
+        Movie movie = new Movie();
+        movie.setTitle("Moana");
+        movieSession.setMovie(movie);
+        movieService.add(movie);
+        movieSessionService.add(movieSession);
```

Comment:
```suggestion
        MovieSession movieSession = new MovieSession();
        Movie movie = new Movie();
        movie.setTitle("Moana");
        movieSession.setMovie(movie);
        movieService.add(movie);
        movieSessionService.add(movieSession);
        User user = authenticationService
                .register("user@gmail.com", "1234");
```
can we do like this?

Reply:
yep, fixed, but can you explain the difference?

Reply:
there is no difference, except that now the checkstyle passes without making the variable final

Reply:
ok)